packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -98,7 +98,7 @@ export type DehydratedUserColumnWidth = [ColumnName, number];
 export type DehydratedUserRowHeight = [number, number];
 
 export type DehydratedSort = {
-  column: ModelIndex;
+  columnName: ColumnName;
```

Comment:
@mofojed Based on your comment, does this mean I should change this type to

`column: ColumnName | ModelIndex` ?